File: programmers/js/new/level3/2xn타일링.js
function solution(n) {
    const dp = Array(n + 1);
    dp[1] = 1;
    dp[2] = 2;
    for (let i = 3; i <= n; ++i) {
        dp[n] = dp[n - 1] + dp[n - 2];
    }
    return dp[n];
}

File: programmers/python/level3/2xn_타일링.py
def solution(n):
    mod = 1000_000_007
    arr = [0 for i in range(n+1)]
    arr[1] = 1
    arr[2] = 2
    for i in range(3, n+1):
        arr[i] = (arr[i-1] + arr[i-2]) % mod
    return arr[n]